ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -102,13 +102,15 @@ def main(argv=sys.argv[1:]):
     report = {}
     for filename in files:
         report[filename] = []
-    for error in root.findall('error'):
-        filename = error.get('file')
+    errors = root.find('errors')
+    for error in errors:
```

Comment:
Since `errors` is only used once this could stay in one line as before: `for error in root.find('errors'):`